src/structures/SkyBlock/SkyblockInventoryItem.js

```diff
@@ -40,7 +40,7 @@ class SkyblockInventoryItem {
      * Hexadecimal color code of armor
      * @type {string|null}
      */
-    this.color = data.tag.display.color ? `#${data.tag.display.color.toString(16)}` : null;
+    this.color = data.tag.ExtraAttributes.color ?? data.tag.display.color;
```

Comment:
u forgot `?? null` at the end so it's `data.tag.ExtraAttributes.color ?? data.tag.display.color ? null;`